src/views/Players/AddPlayers.jsx

```diff
@@ -0,0 +1,49 @@
+import { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import '../../App.css';
+import { createPlayer } from '../../services/players.js';
```

Comment:
make sure to add those blank lines after the import, the visual white space makes for easier reading for future you